webapp-django/crashstats/supersearch/jinja2/supersearch/search_results.html

```diff
@@ -90,22 +75,14 @@
             <tr>
               <td>{{ loop.index }}</td>
               <td>
-                {% if facet != 'signature' %}
-                  <a href="{{ current_url }}&amp;{{ facet }}=%3D{{ hit.term | urlencode }}" class="term" data-field="{{ facet }}" data-content="{{ hit.term }}" title="Add this term to the search form">{{ hit.term }}</a>
-                {% else %}
-                  <a href="{{ url('signature:signature_report') }}{{ change_query_string(
-                     _no_base=True,
-                     signature=hit.term,
-                     page=None,
-                     _columns=None,
-                     _facets=None
-                     ) }}"
-                     title="Click to leave this search and go to this signature report"
-                  >
-                    {{ hit.term }}
-                  </a>
+                {% if facet == 'signature' %}
+                  {{ signature_link(facet, hit.term) }}
                   &nbsp;
-                  <a href="{{ current_url }}&amp;{{ facet }}=%3D{{ hit.term | urlencode }}" class="term" data-field="{{ facet }}" data-content="{{ hit.term }}" title="Add this term to the search form">Add term</a>
+                  {{ refine_link(facet, hit.term, "Add term") }}
+                {% elif facet in ('product', 'version', 'platform', 'process_type') %}
+                  {{ refine_link(facet, hit.term, use_is_query=False) }}
```

Comment:
I added this section because these are special fields that affect site navigation and the supersearch form.